index.html

```diff
@@ -219,27 +198,26 @@ <h2>Extensions to the <dfn data-cite="!hr-time-2#sec-performance">Performance</d
       methods found on JavaScript arrays.</p>
       <section>
         <h2><dfn>getEntries()</dfn> method</h2>
-        <p>Returns a <a>PerformanceEntryList</a> object returned by
-        <a href="#filter-buffer-by-name-and-type"></a> algorithm with
-        <var>buffer</var> set to <a>relevant performance entry buffer</a>, and
-        <var>name</var> and <var>type</var> set to `null`.</p>
+        <p>Returns a <a>PerformanceEntryList</a> object returned by <a href=
+        "#filter-buffer-by-name-and-type"></a> algorithm with <var>buffer</var>
+        set to <a>relevant performance entry buffer</a>, and <var>name</var>
+        and <var>type</var> set to <code>null</code>.</p>
       </section>
       <section>
         <h2><dfn>getEntriesByType()</dfn> method</h2>
-        <p>Returns a <a>PerformanceEntryList</a> object returned by
-        <a href="#filter-buffer-by-name-and-type"></a> algorithm with
-        <var>buffer</var> set to <a>relevant performance entry buffer</a>,
-        <var>name</var> set to `null`, and <var>type</var> set
-        to `type`.</p>
+        <p>Returns a <a>PerformanceEntryList</a> object returned by <a href=
+        "#filter-buffer-by-name-and-type"></a> algorithm with <var>buffer</var>
+        set to <a>relevant performance entry buffer</a>, <var>name</var> set to
+        <code>null</code>, and <var>type</var> set to `type`.</p>
       </section>
       <section>
         <h2><dfn>getEntriesByName()</dfn> method</h2>
-        <p>Returns a <a>PerformanceEntryList</a> object returned by
-        <a href="#filter-buffer-by-name-and-type"></a> algorithm with
-        <var>buffer</var> set to <a>relevant performance entry buffer</a>,
-        <var>name</var> set to `name`, and <var>type</var> set
-        to `null` if optional `entryType` is omitted, and <var>type</var> set
-        to `type` otherwise.</p>
+        <p>Returns a <a>PerformanceEntryList</a> object returned by <a href=
+        "#filter-buffer-by-name-and-type"></a> algorithm with <var>buffer</var>
+        set to <a>relevant performance entry buffer</a>, <var>name</var> set to
+        `name`, and <var>type</var> set to <code>null</code> if optional
```

Comment:
Why are we fixing from markdown backtick notation to `<code>` in some places, but not all?

Reply:
I think these should be `<a>`s (which ReSpec can the auto link and code). Doing the `<a>` requires a little more work tho.